src/ui-material/src/wrappers/form-field.ts

```diff
@@ -12,7 +12,7 @@ import { takeUntil } from 'rxjs/operator/takeUntil';
     <!-- fix https://github.com/angular/material2/pull/7083 by setting width to 100% -->
     <mat-form-field [floatLabel]="to.floatLabel" [style.width]="'100%'">
       <ng-container #fieldComponent></ng-container>
-      <mat-label *ngIf="to.label && field.type !== 'checkbox'">{{ to.label }}</mat-label>
+      <mat-label *ngIf="to.label && field.type !== 'checkbox'">{{ to.label }} {{ to.required ? '*' : '' }}</mat-label>
```

Comment:
I just checked material and I noticed they already handle that case using 'mat-form-field-required-marker' so it would be better to follow them.
```html
<mat-label *ngIf="to.label && field.type !== 'checkbox'">
  {{ to.label }}
  <span *ngIf="to.required" class="mat-form-field-required-marker">*</span>
</mat-label>
```